akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -220,4 +220,11 @@ object Sink extends SinkApply {
     new Sink(new AcknowledgeSink(bufferSize, DefaultAttributes.acknowledgeSink, shape("AcknowledgeSink"), timeout))
   }
 
+  /**
+   * Crates a `Sink` that is materialized into `Source` to connect with another flow.
+   * This sink gets backpressure from second flow and does not have internal buffer.
+   */
+  def pipedSink[T](): Sink[T, Source[T, Unit]] =
+    Sink.publisher[T].mapMaterializedValue(Source(_)).named("PipedSink")
```

Comment:
In general good, though it may be worth not dropping down to the Publisher level, as it will be hard to optimise this away during materialization, won't it? I.e. When I'd like to fuse this "sink-source" with the other stages.